ios/FluentUI/Bottom Sheet/BottomCommandingController.swift

```diff
@@ -0,0 +1,592 @@
+            sheetController.view.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            sheetController.view.topAnchor.constraint(equalTo: view.topAnchor),
+            heroCommandStack.leadingAnchor.constraint(equalTo: commandStackContainer.leadingAnchor, constant: Constants.BottomSheet.heroStackLeadingTrailingMargin),
+            heroCommandStack.trailingAnchor.constraint(equalTo: commandStackContainer.trailingAnchor, constant: -Constants.BottomSheet.heroStackLeadingTrailingMargin),
+            heroCommandStack.bottomAnchor.constraint(equalTo: commandStackContainer.bottomAnchor, constant: -Constants.BottomSheet.heroStackBottomMargin),
+            heroStackTopConstraint
+        ])
+
+        bottomSheetController = sheetController
+        updateExpandability()
+    }
+
+    private func makeBottomBarByEmbedding(contentView: UIView) -> UIView {
+        let bottomBarView = UIView()
+        bottomBarView.layer.shadowColor = Constants.BottomBar.Shadow.color
+        bottomBarView.layer.shadowOpacity = Constants.BottomBar.Shadow.opacity
+        bottomBarView.layer.shadowRadius = Constants.BottomBar.Shadow.radius
+
+        let roundedCornerView = UIView()
+        roundedCornerView.backgroundColor = Constants.BottomBar.backgroundColor
+        roundedCornerView.translatesAutoresizingMaskIntoConstraints = false
+        roundedCornerView.layer.cornerRadius = Constants.BottomBar.cornerRadius
+        roundedCornerView.layer.cornerCurve = .continuous
+        roundedCornerView.clipsToBounds = true
+
+        bottomBarView.addSubview(roundedCornerView)
+        roundedCornerView.addSubview(contentView)
+
+        NSLayoutConstraint.activate([
+            roundedCornerView.leadingAnchor.constraint(equalTo: bottomBarView.leadingAnchor),
+            roundedCornerView.trailingAnchor.constraint(equalTo: bottomBarView.trailingAnchor),
+            roundedCornerView.topAnchor.constraint(equalTo: bottomBarView.topAnchor),
+            roundedCornerView.bottomAnchor.constraint(equalTo: bottomBarView.bottomAnchor),
+            contentView.leadingAnchor.constraint(equalTo: bottomBarView.leadingAnchor, constant: Constants.BottomBar.heroStackLeadingTrailingMargin),
+            contentView.trailingAnchor.constraint(equalTo: bottomBarView.trailingAnchor, constant: -Constants.BottomBar.heroStackLeadingTrailingMargin),
+            contentView.topAnchor.constraint(equalTo: bottomBarView.topAnchor, constant: Constants.BottomBar.heroStackTopBottomMargin),
+            contentView.bottomAnchor.constraint(equalTo: bottomBarView.bottomAnchor, constant: -Constants.BottomBar.heroStackTopBottomMargin)
+        ])
+
+        return bottomBarView
+    }
+
+    private func makeBottomSheetContent(headerView: UIView, expandedContentView: UIView) -> UIView {
+        let view = UIView()
+        let separator = Separator()
+        separator.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
sidenote: I think separator sets it's intrinsicContentSize to something like "NO_INTRINSIC_SIZE" so I wonder if it even matters if we set this  mask :D

I would still set it for consistency, I just thought that was interesting

Reply:
It does have an autoresizing mask tho, so without this that gets converted to constraints and it clashes with mine.